src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -107,7 +107,9 @@ class ServiceInfo extends React.Component<ServiceDetailsId, ServiceInfoState> {
   }
 
   validationsFor(objectType: string) {
-    return this.state.validations.get(objectType);
+    return this.state.validations && typeof this.state.validations.get === 'function'
```

Comment:
This has a bit of a smell to it. But let's fix this now and cleanup later as tomorrow is the sprint demo.

Reply:
probably related to this: https://github.com/kiali/kiali-ui/pull/360#issuecomment-391791684